src/backend/db/geese-features.js

```diff
@@ -0,0 +1,74 @@
+import * as R from 'ramda';
+
+const getFeatures = (db) => () =>
+  db('geese-features')
+    .then((features) => {
+      return { success: 'Y', features };
+    })
+    .catch((error) => {
+      return { success: 'N', features: [] };
+    });
+
+const getFeatureById = (db) => (id) =>
+  db('geese-features')
+    .where({ id })
+    .then((res) => {
+      return R.isEmpty(res) ? res : res[0];
```

Comment:
[1] can simply return res here and rely on the logic defined in `src/backend/api/geese-features/get-feature.js`.

(See other comment marked as [1])